sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeContentFromFile.cs

```diff
@@ -39,6 +39,16 @@ public async Task RecognizeContentFromFile()
                     FormLine line = page.Lines[i];
                     Console.WriteLine($"  Line {i} has {line.Words.Count} {(line.Words.Count == 1 ? "word" : "words")}, and text: '{line.Text}'.");
 
+                    if (line.Appearance != null)
+                    {
+                        // Check the style and style confidence to see if text is handwritten.
+                        // Note that value '0.8' is used as an example.
+                        if (line.Appearance.Style.Name == TextStyleName.Handwriting && line.Appearance.Style.Confidence > 0.8)
+                        {
+                            Console.WriteLine("The text is handwritten");
```

Comment:
Not sure if it's worth printing out the text that might be written like 

Console.WriteLine("The text {line.Text} is handwritten");

Reply:
In line 40 I print the text with he number of words, so this is additive to the printed block